addons/mail/models/mail_message.py

```diff
@@ -1085,6 +1087,21 @@ def _notify_message_notification_update(self):
         ]
         self.env['bus.bus']._sendmany(updates)
 
+    def delete_attachment_and_notify(self, attachments):
```

Comment:
method should be private (always make it so by default) as we don't want to call it externally